src/data/faq.json

```diff
@@ -295,7 +295,8 @@
                         "anchor": "interoperability_countries",
                         "active": true,
                         "textblock": [
-                            "Together with the European Commission, an infrastructure for secure information exchange has been developed, known as the \"Interoperability Gateway\". This gateway enables the Corona Warn App to operate across national borders and to receive potential warning messages abroad.",
+                            "Together with the European Commission, an infrastructure for secure information exchange has been developed, known as the \"Interoperability Gateway\". This gateway enables the Corona Warn App to operate across national borders and to receive potential warning messages abroad. Detailed information about the Gateway is available from the European Commission in <a href='https://ec.europa.eu/commission/presscorner/detail/en/qanda_20_1905#gateway'>Coronavirus: EU interoperability gateway for contact tracing and warning apps – Questions and Answers</a>.",
+                            "To view more information in the app, tap EXPOSURE LOGGING ACTIVE (or STOPPED), then tap Transnational Exposure Logging, Participating Countries.",
```

Comment:
I think it is better to have these instructions in a list format similar to https://www.coronawarn.app/en/faq/#max_power_save

Reply:
Edited it, did you mean like this? Tested the changes via the Chrome Dev Tools (-: